pkg/controller/configmap_control.go

```diff
@@ -81,7 +89,7 @@ func (cc *realConfigMapControl) UpdateConfigMap(owner runtime.Object, cm *corev1
 			return nil
 		}
 
-		if updated, err := cc.cmLister.ConfigMaps(cm.Namespace).Get(cmName); err != nil {
+		if updated, err := cc.kubeCli.CoreV1().ConfigMaps(cm.Namespace).Get(cmName, metav1.GetOptions{}); err != nil {
```

Comment:
I think it is not necessary to operate kube-apiserver directly, refer [here](url).

Reply:
Cache the configmaps will increase the memory usage a lot.